test/init_test.go

```diff
@@ -50,6 +51,23 @@ func init() {
 	flag.BoolVar(&skipRootUserTests, "skipRootUserTests", false, "Skip tests that require root user")
 }
 
+func setupWithOverrides(t *testing.T, overrides *clientcmd.ConfigOverrides, fn ...func(*testing.T, *clients, string)) (*clients, string) {
```

Comment:
similar question here, wondering if we can avoid some duplication b/w this and setup